CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - You can now cite XAVIER with the DOI [10.5281/zenodo.12727315](https://doi.org/10.5281/zenodo.12727315). (#88, @kelly-sovacool)
 - Minor documentation improvements. (#92, @kelly-sovacool)
+- Minor documentation rendering improvements (#93, @samarth8392)
```

Comment:
We can consolidate these entries to one line:

```suggestion
- Minor documentation improvements. (#92, @kelly-sovacool; #93, @samarth8392)
```

Reply:
Sorry didn't see your comment before I pulled the trigger on merging the PR. I have other issues to work on, will change it in the next PR.

Reply:
@samarth8392 no worries!